python/lsst/ts/mtaos/mtaos.py

```diff
@@ -1006,6 +1018,24 @@ async def run_closed_loop(self):
                         self.log.info(f"Visit {visit_id} already processed, skipping.")
                         continue
 
+                    filter_label, elevation = await self.model.get_image_info(
+                        visit_id,
+                        self.camera_name,
+                    )
+
+                    rotation_angle = float(
+                        np.mean(np.array(self.image_rotator[image_in_oods.obsid]))
+                    )
+
+                    if (
+                        np.abs(rotation_angle - self.current_rotator_position)
```

Comment:
There's a change that you could get here and either (or both) `self.current_rotator_position` or `self.current_elevation_position` are `None`. Make sure you handle this condition here.